peps/pep-0696.rst

```diff
@@ -253,14 +242,14 @@ Using a type parameter from an outer scope as a default is not supported.
 Bound Rules
 ~~~~~~~~~~~
 
-``T2``'s bound must be a subtype of ``T1``'s bound.
+``T1``'s bound must be a subtype of ``T2``'s bound.
```

Comment:
It's been a while since I've had to think about this, but I don't think this is correct.
The bound should be able to get more specific not less, making it less seems, at least to me, unsound.

The way I've reasoned with this is with:
```py
class _[T1: float, T2: int = T1, T3: bool = T2]:
    ...
```
Would you agree that this should be fine?

Reply:
Doesn't this allow a TypeVar's bound to be violated? Like this:
```
class C[T1: float, T2: int = T1]: ...
reveal_type(C[float]())  # This is C[float, float], but T2=float violates T2's upper bound of int
```

Reply:
Ah, I see this makes sense